easybuild/easyconfigs/i/IMB/IMB-4.1-intel-2017.02.eb

```diff
@@ -0,0 +1,55 @@
+# Built with EasyBuild version 3.1.1 on 2017-04-21_13-11-20
+easyblock = 'MakeCp'
+
+name = 'IMB'
+version = '4.1'
+
+homepage = 'https://software.intel.com/en-us/articles/intel-mpi-benchmarks'
+description = """The Intel MPI Benchmarks perform a set of MPI performance measurements for point-to-point and
+ global communication operations for a range of message sizes"""
+
+toolchain = {'name': 'intel', 'version': '2017.02'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://software.intel.com/sites/default/files/managed/34/aa/']
+sources = ['%(name)s_%(version)s.tgz']
+
+prebuildopts = "cd src && "
+buildopts = "all"
```

Comment:
is this strictly required? it's not there in #4517...

Is that why `IMB-IO` is being listed here in the sanity check, but not in #4517? If so, we should use `all` there too and enhance the sanity check?